README.md

```diff
@@ -0,0 +1,141 @@
+<p align="center">
+    <img alt="Plone Logo" width="200px" src="https://raw.githubusercontent.com/plone/.github/main/plone-logo.png">
+</p>
+
+<h1 align="center">
+  Plone
+</h1>
+
+<div align="center">
+
+[![PyPI](https://img.shields.io/pypi/v/plone)](https://pypi.org/project/plone/)
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/plone)](https://pypi.org/project/plone/)
+[![PyPI - Wheel](https://img.shields.io/pypi/wheel/plone)](https://pypi.org/project/plone/)
+[![PyPI - License](https://img.shields.io/pypi/l/plone)](https://pypi.org/project/plone/)
+[![PyPI - Status](https://img.shields.io/pypi/status/plone)](https://pypi.org/project/plone/)
+
+[![GitHub contributors](https://img.shields.io/github/contributors/plone/Products.CMFPlone)](https://github.com/plone/Products.CMFPlone)
+![GitHub Repo stars](https://img.shields.io/github/stars/plone/Plone?style=flat-square)
+
+</div>
+
+Plone is a mature, secure, and user-friendly Content Management System (CMS).
```

Comment:
English is weird. The term should be lower-cased, yet the acronym is upper-cased.

```suggestion
Plone is a mature, secure, and user-friendly content management system (CMS).
```